src/p5.dance.js

```diff
@@ -966,7 +971,7 @@ module.exports = class DanceParty {
         sprite.rotation = 0;
       });
     } else {
-      throw new Error('Unexpected format: ' + format);
+      console.log('Unexpected layout format: ' + format);
```

Comment:
Just noting that if OpenAI responds with an invalid costume, we ignore and will not display dancers. However, the generated code and explanation will still reference the hallucinogenic response, in the example below, the costume 'rabbit'.
<img width="816" alt="Screen Shot 2023-10-02 at 3 26 10 PM" src="https://github.com/code-dot-org/dance-party/assets/107423305/1d90dea1-798c-4464-9d8f-d408e4d2ed4f">

Reply:
Yes, it's a good point.  I think we should work out a plan for detecting this situation and explaining it as a teachable moment!